src/infrastructure/note.repository.ts

```diff
@@ -30,16 +58,25 @@ export default class NoteRepository implements NoteRepositoryInterface {
     const noteData = await this.noteStorage.getNoteById(id);
 
     /**
-     * If note data not exists
+     * If note data in storage exists
      */
-    if (!noteData) {
-      return null;
+    if (noteData) {
+      return {
```

Comment:
you can just return `noteData` itself since we agreed that a Storage stores Note objects

Reply:
done